std_msgs/Int32 glove_id
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity
geometry_msgs/Vector3 acceleration
geometry_msgs/Quaternion rotation
bool[10] buttons
# 0 => config button
# 1-4 => action buttons
# 5 => takeoff/land
# 6 => slide
# 7 => emergency abort
# 8 => prev drone
# 9 => next drone
